terraform/monitoring/panels/app/publishing_workers_errors.libsonnet

```diff
@@ -0,0 +1,19 @@
+local grafana   = import '../../grafonnet-lib/grafana.libsonnet';
+local defaults  = import '../../grafonnet-lib/defaults.libsonnet';
+
+local panels    = grafana.panels;
+local targets   = grafana.targets;
+
+{
+  new(ds, vars)::
+    panels.timeseries(
+      title       = 'Publishing worker tasks errors count',
+      datasource  = ds.prometheus,
+    )
+    .configure(defaults.configuration.timeseries)
+    .addTarget(targets.prometheus(
+      datasource  = ds.prometheus,
+      expr        = 'sum(rate(publishing_worker_errors_total{}[$__rate_interval]))',
```

Comment:
Probably nice to add an alarm for this? Later PR is fine. Example here: https://github.com/WalletConnect/notify-server/pull/191

Reply:
Yep, should be a follow-up #196 when all metrics will be alive, to adjust thresholds.